lib/runTx.js

```diff
@@ -140,6 +140,7 @@ module.exports = function (opts, cb) {
       // process any gas refund
       var gasRefund = results.vm.gasRefund
       if (gasRefund) {
+        results.gasRefund = gasRefund;
```

Comment:
I think it would be better if ``results.vm.gasRefund`` is always assigned to ``gasRefund`` than this always has a value, being it ``BN(0)`` or something else.

Reply:
Sorry, I'm not following for some reason.

Do you mean something along the lines of:
`var gasRefund = results.vm.gasRefund || BN(0)`

or are you looking for me to make sure that `results.vm.gasRefund` has a value?

Reply:
Along the lines of the first assignment, but you can leave the ``|| BN(0)`` part, ``results.vm.gasRefund`` is already initialized with 0 I think (you might want to cross-check this).